portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,65 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+      <h1 id="name_heading"><b>PREM RAJ OLI</b></h1>
+      <img src="/images/pp1.jpg" alt="profile image" style="width:700px;height:700px">
+      <p>
+          Hi, there! This is Prem. I am a Computer Science student and I love cooking, enjoy 
+          spicy foods, and listen to rap music. To know more about me read through my profile below.
+      </p>
+      <h2>PROFILE</h2>
+      <p>
+          I am orginally from Banke district of Nepal. After completing my high school in 2019, I got 
+          admission from Howard University. I had to complete first year online while staying in home 
+          country due to Covid-19 and I arrived in USA in 2021. Currently, I am a rising junior pursuing my bachelor's degree in Computer 
+          Science at Howard University in Washington, D.C. I am an enthusiastic Computer Science Student 
+          interested in Software develpoment, Machine Learning, and Artificial Intelligence.
+      </p>
+      <h2>PROJECTS</h2>
+      <dl>
+          <dt>Online Shopping Application</dt>
+          <dd>
+            Built a database based shopping cart web application able to handle administrator
+            and customer roles.
+          </dd>
+          <dd>
+            Administrator role: allowed to manage the products, customers and stores, and access
+            dashboard to identify the top products
+          </dd>
+          <dd>
+            Customer role: allowed review, add, delete, or edit the number of products in the 
+            shopping cart, and submit orders
+          </dd>
+        <dt>Satellite Imaging w/ NGA</dt>
+        <dd>
+            Built an object Detecting program using Tensor-FLow, Keras API, and MASK-RCNN 
+            algorithm in Python
+        </dd>
+        <dd>
+            The goal of this project was to detect the object from satellite images with probability 
+            percentage by using pre-trained datatsets and training the algorithm on custom datasets
+        </dd>
+        <dt>College Record System</dt>
+        <dd>
+            Developed a GUI Interface that ask the user for personal informations, such as name, 
+            address,and email, and save the record in a csv file
+        </dd>
+        <dd>
+            Used Python's Tkinter library for GUI Interface
+        </dd>
+      </dl>
+
+      <!--<p>Click <a href="/hello">here</a> to see content created by server code.</p>
```

Comment:
Instead of commenting this out, it's better to delete it. If you want to recover it later, you can look through your change history in git.